packages/fx-core/templates/plugins/resource/cicd/azdo-pipelines/azure-pipeline-cd.yml

```diff
@@ -0,0 +1,34 @@
+trigger:
+  - main
+
+pool:
+  vmImage: ubuntu-latest
+
+steps:
+  # Build the project.
+  - task: teams-app-cicd@1
+    inputs:
+      operationType: "Build Teams App"
+  # Run unit test.
+  - task: NodeTool@0
+    inputs:
+      versionSpec: "10.x"
```

Comment:
is it node version? why is 10

Reply:
In our Teams Tookit, we recommend user to use Node 14, so we need to keep aligned node version in our pipeline templates